packages/belos/tpetra/test/Native/CMakeLists.txt

```diff
@@ -92,7 +92,31 @@ TRIBITS_ADD_TEST(
 TRIBITS_ADD_TEST(
   Tpetra_Native_solvers_nonsymm_matrix
   NAME Tpetra_Native_GMRES_SINGLE_REDUCE_nonsymm_matrix
-  ARGS "--solver=\"TPETRA GMRES SINGLE REDUCE\" --restartLength=10"
+  ARGS "--solver=\"TPETRA GMRES SINGLE REDUCE\" --restartLength=10 --noRitzValues"
```

Comment:
Does there exist a test in this file that does not have the --noRitzValues option?  Perhaps we should still test that case?

Reply:
@iyamazaki What about this one?  Do we need to add a test without the --noRitzValues flag?

Reply:
We don't usually need Newton basis for single-reduce GMRES, but you are right @jennloe  we should test that option. I added it to the test.